### ISSUE TITLE
Highlight in-screen updates

### ISSUE THREAD

**Issue by zdmc23** [DiscipleTools/disciple-tools-mobile-app#76]
*As a* User,
*I want to* see a visual cue that an in-screen update has occurred (everywhere applicable)
*in order to* understand that new data is available

- [ ] Visual cue indicates new data is available to user (where applicable)

![ss3](https://user-images.githubusercontent.com/191707/67646666-9ca93b80-f905-11e9-9957-6e7da02ac157.png)

**Comment by zdmc23** [comment]
There was an initial implementation on this, but that prompted a significant amount of discussion on our Slack channel, so we should continue that discussion here...

**Comment by mikeallbutt** [comment]
Like is Slack, when a new message is added to a thread, it is announced by changing the background colour for a brief time. This notification doesn't need to be too pronounced. Maybe a last updated date on a contact card and also in the list will help. And if the update was recent then the text (or and icon) for that record could indicate that. (Flames icon to show the data is hot!)

**Comment by cairocoder01** [comment]
We need to define what "everywhere applicable" actually means. What things do we want indication that they've been updated? My initial understanding was just for comments on a contact/group. The initial implementation did it for all fields on a contact, which didn't seem helpful. I would say that it might be helpful on the contact/group list pages to show contacts that have had any activity (updated date) since last viewing it (need to track last time current user has view that contact in local storage).

**Comment by zdmc23** [comment]
Yes, let's narrow the scope to the Comments/Activities list, and then expand from there (if commonly requested by users). 

In-App Notifications could serve for notice of new Group(s), and Push Notifications for new Contact(s) that you're personally associated with.  Those kind of negate the need for highlighting

**Comment by cairocoder01** [comment]
Sounds great.

## Requirements ##
- [ ] Track `last_viewed` date for each contact in local storage
  - When user opens contact detail screen, set `contact.last_viewed` to now
- [ ] On contact comments/activity tab, highlight any comments/activity with a creation date after `contact.last_viewed`
- [ ] Track `last_viewed` date for each group in local storage
  - When user opens group detail screen, set `group.last_viewed` to now
- [ ] On group comments/activity tab, highlight any comments/activity with a creation date after `group.last_viewed`

## Design ##
Box color: #ADBCCB
Activity text color: #8C8C8C
![image](https://user-images.githubusercontent.com/44169599/67769271-d476c700-fa5c-11e9-9878-7af6e1b58a09.png)

**Comment by mikeallbutt** [comment]
I like what @cairocoder01 suggests too. Track last_viewed date will help. Can this info be stored and used by the DT website too. (this would be a good feature to implement on the website too IMO)
